framework/test/jest/config/setup.js

```diff
@@ -15,6 +15,8 @@
  */
 
 require('jest-extended');
-require('jest-chain');
+
```

Comment:
probably this will be cleared once the other PR merged in right?

Reply:
Yeap